website/content/docs/commands/operator/secure-variables/keyring-remove.mdx

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+page_title: 'Commands: operator secure-variables keyring remove'
+description: |
+  Remove an unused encryption key
+---
+
+# Command: operator secure-variables keyring remove
+
+The `operator secure-variables keyring remove` command removes an
+encryption key from the cluster. This operation may only be performed
+on keys that are not the active key.
```

Comment:
Is `active` the same as primary?

Reply:
Good question! The gossip keyring docs use the terminology "primary" but it's honestly a bit of misnomer because "primary" implies "secondary", etc. whereas there's really just one active key and the rest are inactive. For the secure variables keyring, which is what this command touches, we're using Active vs Inactive (and Rekeying and Deprecated). 

In Nomad 1.5.0-ish we'll probably drop the gossip key encryption commands entirely and make these operations owned by Nomad automagically, because of https://github.com/hashicorp/nomad/issues/13381, so I didn't want to go messing around with the existing gossip keyring docs too much.